specification/cognitiveservices/data-plane/InkRecognizer/readme.go.md

```diff
@@ -0,0 +1,33 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: inkrecognizer
+  clear-output-folder: true
+
+  directive:
+  # Replace the default client constructor with the one that enables TLS renegotiation.
+  - from: client.go
+    where: $
+    transform: >-
+      return $.
+        replace( /"context"/g, "\"context\"\n\"crypto/tls\"" ).
+        replace( /autorest\.NewClientWithUserAgent\(UserAgent\(\)\)/g, "autorest.NewClientWithOptions(autorest.ClientOptions{UserAgent: UserAgent(), Renegotiation: tls.RenegotiateFreelyAsClient})" )
+```
+
+```yaml $(go multi-api)
+batch:
+  - tag: release_1_0
```

Comment:
The value `release_1_0` doesn't match the actual tag `package-1.0`, can you please fix?  I believe other config files have the same issue.

Reply:
@sarangan12 FYI this same issue is the cause of failure for the Ruby SDK build

Reply:
@jhendrixMSFT  I've fixed this in readme and readme.go.md. I don't see it in the ruby file though. Am I missing something?

Reply:
Ruby is also `release_1_0` but now that you updated the README they all match.